### ISSUE TITLE
more... links for facets need facet name

### ISSUE THREAD

**Issue by jvine** [sul-dlss/SearchWorks#345]
For facets that have more values than are shown by default:

a. the "more" link has disappeared
b. the link should be structured like this:

```
<a ...>more <span class="sr-only">languages</span> <i class="fa fa-angle-double-right"></i></a>
```

where "languages" is whatever the facet name is - "more authors" "more topics" etc. (if it's any help, I think you can just take the facet name and add s - any this doesn't work for won't have more links anyway.)
